pkg/cmd/rules_engine/create/create.go

```diff
@@ -0,0 +1,122 @@
+package create
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/MakeNowJust/heredoc"
+
+	msg "github.com/aziontech/azion-cli/messages/rules_engine"
+	api "github.com/aziontech/azion-cli/pkg/api/edge_applications"
+
+	"github.com/aziontech/azion-cli/pkg/cmdutil"
+	"github.com/aziontech/azion-cli/utils"
+	"github.com/spf13/cobra"
+)
+
+type Fields struct {
+	ApplicationID int64
+	Phase         string
+	Path          string
+}
+
+func NewCmd(f *cmdutil.Factory) *cobra.Command {
+	fields := &Fields{}
+
+	cmd := &cobra.Command{
+		Use:           msg.RulesEngineCreateUsage,
+		Short:         msg.RulesEngineCreateShortDescription,
+		Long:          msg.RulesEngineCreateLongDescription,
+		SilenceUsage:  true,
+		SilenceErrors: true,
+		Example: heredoc.Doc(`
+        `),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if !cmd.Flags().Changed("application-id") || !cmd.Flags().Changed("phase") {
+				return msg.ErrorMandatoryCreateFlags
+			}
+
+			request := api.CreateRulesEngineRequest{}
+			if cmd.Flags().Changed("in") {
```

Comment:
We can remove this check. --in flag now is mandatory, so this check should be with the other ones above

Reply:
I agree